src/components/account/fission.ts

```diff
@@ -300,6 +352,32 @@ export async function did(
   const root = Array.from(rootIssuers.values())[0]
   if (!root) throw new Error("Expected a root issuer to be found")
   return root
+
+  // TODO: Get public key from fission server generated key pair
+  //       -> issuer from fission-server issued UCAN
+}
+
+export async function hasSufficientAuthority(
```

Comment:
It depends on the implementation. The idea is that you'd use a different account implementation based on the authority needed. There is an implementation for a Fission dev account which checks for capabilities to update the volume and register + publish apps. But there's also implementations for app accounts and delegated access, although I'm guessing these could be the same seeing that they only capability they have is to update their volume.

> Seems like maybe the latter because we still want to load volumes in apps without full account access

I should also mention that this method is only for the account capabilities, not file system capabilities and access keys, so loading volumes is independent of this.